linera-service/src/cli_wrappers/wallet.rs

```diff
@@ -1204,6 +1206,19 @@ impl NodeService {
         Ok(serde_json::from_value(committees)?)
     }
 
+    pub async fn events_from_index(
+        &self,
+        chain_id: &ChainId,
+        stream_id: &StreamId,
+        start_index: u32,
+    ) -> Result<Vec<IndexAndEvent>> {
+        let query = format!("query {{ eventsFromIndex(chainId: \"{chain_id}\", streamId: {}, startIndex: {start_index}) {{ index event }} }}", stream_id.to_value());
```

Comment:
(Maybe break the string into multiple lines and format it a bit.)

Reply:
Ok, done.